stacks/apps/feeder.yml

```diff
@@ -508,24 +491,14 @@ Resources:
               Value: !Ref kWebApplicationPort
             - Name: PORTER_SNS_TOPIC
               Value: !Ref PorterJobExecutionSnsTopicArn
+            - Name: POSTGRES_HOST
+              Value: !Ref SharedAuroraPostgresqlEndpoint
+            - Name: POSTGRES_PORT
+              Value: !Ref SharedAuroraPostgresqlPort
+            - Name: RAILS_ENV
+              Value: !Ref EnvironmentTypeLowercase
             - Name: PRX_HOST
               Value: !Ref BetaHostname
-            - Name: RAILS_SERVE_STATIC_FILES
-              Value: enabled
-            - Name: WORKER_PAUSE
-              Value: "30"
-            - Name: WORKER_DATABASE_POOL_SIZE
-              Value: !If [IsProduction, 16, 5]
-            - Name: WORKER_COUNT
```

Comment:
Don't we need this still? I see it in the `config/shoryuken.yml`:
```
concurrency: <%= ENV['WORKER_COUNT'] || 25 %>
```

Reply:
Yep - still defined in the worker task def.  Just not the web.